src/phrases/phrases-nb.ts

```diff
@@ -1275,7 +1275,7 @@ const phrases = {
     articleDisclaimer: {
       title: "Informasjon om tilgjengelighet for hele artikkelen",
       description:
-        "Skriv inn hva som er utfordrende med innholdet i artikkelen. Du kan også legge til lenker til alternativt innhold. Det er utarbeidet maler for dette. Sjekk disse før du tar i bruk komponenten. ",
+        "Skriv inn hva som er utfordrende med innholdet i artikkelen. Du kan også legge til lenker til alternativt innhold. Det er utarbeidet maler for dette. Sjekk disse <0>her</0> før du tar i bruk komponenten. ",
```

Comment:
Kan kanskje bruke {{}} for å sette inn urlen?

Reply:
Nei, trenger å sette inn Safelink-komponent for at oppførselen skal bli riktig😅